app/assets/javascripts/_practice_card_utilities.es6

```diff
@@ -74,6 +72,47 @@ function replacePlaceholderWithImage(imageUrl, practiceId, practiceName) {
     });
 }
 
+function replaceSearchPageImagePlaceholders(practiceEls) {
+    $('.dm-search-result').each(function() {
+        const $resultElement = $(this);
+        const placeholder = $resultElement.find('.search-result-img-container');
+
+        if (placeholder.find('img').length > 0) {
+            return;
+        }
+
+        const practiceId = placeholder.attr('data-practice-id');
+        const imagePath = placeholder.attr('data-practice-image');
+        const practiceName = placeholder.attr('data-practice-name');
+
+        if (practiceId && imagePath) {
+            fetchSignedResource(imagePath).then(signedUrl => {
+
+                replaceSearchResultPlaceholderImage(signedUrl, practiceId, practiceName).then(updatedElement => {
+                    practiceEls[practiceId] = updatedElement;
+                });
+            });
+        }
+    });
+}
+
+function replaceSearchResultPlaceholderImage(imageUrl, practiceId, practiceName) {
+    return new Promise((resolve, reject) => {
+        loadImage(imageUrl, function(loadedImageSrc) {
+            const imgElement = $('<img>')
+                .attr('data-resource-id', practiceId)
+                .attr('src', loadedImageSrc)
+                .attr('alt', practiceName + ' Search Result Image')
```

Comment:
I'd suggest we set these as `alt=""` if there's no alt text available in `practice.main_display_image_alt_text`. I know this is pre-existing behavior, but I don't think it's worth perpetuating.